<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureRvsTDialog</class>
 <widget class="QDialog" name="ConfigureRvsTDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure 1/R vs 1000/T Measurement</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QGroupBox" name="measureTypeGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Measure Type</string>
   </property>
   <widget class="QRadioButton" name="radioButtonSourceI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Source I Measure V</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSourceV">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Source V Measure I</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="testValueEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnits">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSourceVal">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelCompliance">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Compliance</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="complianceValueEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelComplianceUnits">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMeasureInterval">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>90</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Measurement Interval</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="measureIntervalEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="TemperatureGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Temperature Control</string>
   </property>
   <widget class="QLabel" name="labelTStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>T Start</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTEnd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>T End</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="TStartEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>28</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="TEndEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnitsT">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemperatureRate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>T Rate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="TRateEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>59</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnitsTime">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>K/min</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MaxTimeToTStartEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelReachingTStart">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Time for reaching Starting T</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStabilizationT">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>121</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time for Stabilizing Temperature </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>121</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TimeForStabilizingTStartEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>121</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="SampleGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Sample Information</string>
   </property>
   <widget class="QPlainTextEdit" name="sampleInformationEdit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>30</y>
      <width>385</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>630</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="outFileGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Output File</string>
   </property>
   <widget class="QLineEdit" name="outFileEdit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>59</y>
      <width>385</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="outFilePathButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>28</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="outPathEdit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>28</y>
      <width>350</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>630</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="MonochromatorGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>411</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Monochromator</string>
   </property>
   <widget class="QRadioButton" name="radioButtonGrating1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>24</y>
      <width>112</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Grating #1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonGrating2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>54</y>
      <width>112</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Grating #2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWavelength">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>54</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Wavelength</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="WavelengthEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>54</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>54</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>nm</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
